supervisor/conf/config-1.yml

```diff
@@ -0,0 +1,85 @@
+# Connection details for the OpAMP server.
+opampServer:
+    endpoint: https://localhost:1729
```

Comment:
OpAMP is WebSocket, so needs `wss:` scheme:
```suggestion
    endpoint: wss://localhost:1729
```

Reply:
My bad. Will fix this.